rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/tools/command/StopAllSubCommand.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.rocketmq.connect.runtime.tools.command;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.apache.rocketmq.connect.runtime.rest.RestSender;
+import org.apache.rocketmq.tools.command.SubCommandException;
+
+public class StopAllSubCommand implements SubCommand {
+    @Override
+    public String commandName() {
+        return "stopAll";
+    }
+
+    @Override
+    public String commandDesc() {
+        return "Stop and delete all Connectors and all configuration information";
+    }
+
+    @Override
+    public Options buildCommandlineOptions(Options options) {
+        return options;
+    }
+
+    @Override
+    public void execute(CommandLine commandLine, Options options) throws SubCommandException {
+        try {
+            String url = "http://localhost:8081/connectors/" + commandName();
```

Comment:
Why this url is hardcoded in code ~

Reply:
I have avoid hardcoded by adding the configuration file